{{ cookiecutter.repo_name }}/.gitignore

```diff
@@ -120,7 +120,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-.env
+#.env
```

Comment:
As before, should this be commented or removed?

Reply:
Done.